docs/typescript.mdx

```diff
@@ -39,11 +39,20 @@ To make the css prop work with pure TypeScript (without babel plugin) you need t
 
 ```tsx
 /** @jsx jsx */
-import { css, jsx } from '@emotion/core'
-```
+import { jsx } from '@emotion/core'
 
+<div css={{ background: 'black' }} />
+```
 As a result you may be not able to use react fragment shorthand syntax - `<></>`, but still you can use `<Fragment></Fragment>`.
 
+You can still use the css helper and pass the className yourself.
+
+```tsx
+import { css } from '@emotion/core'
+
+<div className={css({ background: 'black' })} />
```

Comment:
this won't work

Either use `css` from `emotion` package or pass the resulting opaque object to `css` prop instead of the `className` (depends on the intention of this added section)

Reply:
Nice, have updated this and added a comment about importing from the correct location.